src/Razor/src/Microsoft.VisualStudio.LanguageServerClient.Razor/CompletionItemExtensions.cs

```diff
@@ -0,0 +1,64 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using Microsoft.CodeAnalysis.Razor.Completion;
+using Microsoft.VisualStudio.LanguageServer.Protocol;
+using Newtonsoft.Json.Linq;
+
+namespace Microsoft.VisualStudio.LanguageServerClient.Razor
+{
+    internal static class CompletionItemExtensions
```

Comment:
This is somewhat a mirror of https://github.com/dotnet/aspnetcore-tooling/blob/master/src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/Completion/CompletionItemExtensions.cs#L11. Needed this because this `CompletionItem` comes from VS Client protocol and that one comes from OmniSharp.